models/enterprise_sonic/snmp/deleted_example_02.txt

```diff
@@ -1,23 +1,24 @@
 # Using deleted
-
-# Before state:
-# -------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# Before state:
+#---------------
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# show running-configuration | grep snmp
 #
+# snmp-server user user1 group group-lab encrypted auth
+# md5 auth-password
+# U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+# priv aes-128 priv-password U2FsdGVkX1/Xs+ffZvdV9YzfyGHgIJ+zkLRPfF3/WgYIE1S4Ribvbzhu5chpHHI7ooCBpcVxYZotAXDzgetxvQ==
+# snmp-server group group-floor2 v3 priv read r_view write w_view notify n_view
+# snmp-server community comm1 group group-lab
 
-- name: Delete all SNMP communities
+- name: Delete all SNMP server configuration
```

Comment:
Sorry for requesting a "delete all" example. I found out that, unlike most of our configuration types, it isn't supported for SNMP server configuration. Instead, please change this example to show deletion of all users when there are multiple users. (The same sort of "section" deletion is supported for the other main sections of SNMP server configuration, but one example is sufficient to illustrate this.)